README.md

```diff
@@ -1,13 +1,191 @@
+usual. It is also not intended to be used as a dependency by consuming
+libraries. Consuming library code should use the
+[array-api-compat](https://github.com/data-apis/array-api-compat) package to
+support the array API. Rather, it is intended to be used in the test suites of
+consuming libraries to test their array API usage.
+
+## Install
+
+`array-api-strict` is available on both
+[PyPI](https://pypi.org/project/array-api-strict/)
+
+```
+python -m pip install array-api-strict
+```
+
+and [Conda-forge](https://anaconda.org/conda-forge/array-api-strict)
+
+```
+conda install --channel conda-forge array-api-strict
+```
+
+array-api-strict supports NumPy 1.26 and (the upcoming) NumPy 2.0.
+
+## Rationale
+
+The array API has many functions and behaviors that are required to be
+implemented by conforming libraries, but it does not, in most cases, disallow
+implementing additional functions, keyword arguments, and behaviors that
+aren't explicitly required by the standard.
+
+However, this poses a problem for consumers of the array API, as they may
+accidentally use a function or rely on a behavior which just happens to be
+implemented in every array library they test against (e.g., NumPy and
+PyTorch), but isn't required by the standard and may not be included in other
+libraries.
+
+array-api-strict solves this problem by providing a strict, minimal
+implementation of the array API standard. Only those functions and behaviors
+that are explicitly *required* by the standard are implemented. For example,
+most NumPy functions accept Python scalars as inputs:
+
+```py
+>>> import numpy as np
+>>> np.sin(0.0)
+0.0
+```
+
+However, the standard only specifies function inputs on `Array` objects. And
+indeed, some libraries, such as PyTorch, do not allow this:
+
+```py
+>>> import torch
+>>> torch.sin(0.0)
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+TypeError: sin(): argument 'input' (position 1) must be Tensor, not float
+```
+
+In array-api-strict, this is also an error:
+
+```py
+>>> import array_api_strict as xp
+>>> xp.sin(0.0)
+Traceback (most recent call last):
+...
+AttributeError: 'float' object has no attribute 'dtype'
+```
+
+Here is an (incomplete) list of the sorts of ways that array-api-strict is
+strict/minimal:
+
+- Only those functions and methods that are [defined in the
+  standard](https://data-apis.org/array-api/latest/API_specification/index.html)
+  are included.
+
+- In those functions, only the keyword-arguments that are defined by the
+  standard are included. All signatures in array-api-strict use
+  [positional-only
+  arguments](https://data-apis.org/array-api/latest/API_specification/function_and_method_signatures.html#function-and-method-signatures).
+  As noted above, only array_api_strict array objects are accepted by
+  functions, except in the places where the standard allows Python scalars
+  (i.e., functions to not automatically call `asarray` on their inputs).
```

Comment:
this sentence reads slightly awkward to me. maybe it's a typo (to not -> do not)?